src/test-helpers/queries/getSubthoughtsByContext.ts

```diff
@@ -0,0 +1,11 @@
+import { screen } from '@testing-library/dom'
+import contextToPath from '../../selectors/contextToPath'
+import store from '../../stores/app'
+
+/** Get a list of subthoughts of a thought with the context. */
+export function getSubthoughtsByContext(context: string[]) {
```

Comment:
Technically this returns all descendants.

Reply:
Ahh that's a good point 🤔

Reply:
Either I get rename the function or update it to only get it to a certain character limit (so it is only one level)

Reply:
I already merged, but I'm open to renaming it. 

We don't really want to add a dependency on the id character length.